pt_os_web_portal/miniscreen_onboarding_assistant/pages.py

```diff
@@ -281,7 +283,10 @@ class CarryOnPage(PageBase):
     def __init__(self, size, mode, interval):
         super().__init__(type=Page.CARRY_ON, size=size, mode=mode, interval=interval)
         self.text = "You've started the onboarding!\nContinue in your browser..."
-        self.visible = False
+        self.visible = (
```

Comment:
Rather than getting it from here, wouldn't it make more sense to have the application parse the state, clear it from the state file, and then set the visibility of this page?